### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/gnucash/gnucash/) for GnuCash/Program.

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -1661,37 +1659,36 @@ msgstr "Vous devez sélectionner un rapport à exécuter."
...
 msgid ""
 "A saved report configuration with this name already exists, please choose "
 "another name."
 msgstr ""
-"Un rapport personnalisé avec ce nom existe déja. Veuillez choisir un autre "
+"Un rapport personnalisé avec ce nom existe déjà. Veuillez choisir un autre "
 "nom."
 
 #: gnucash/gnome/dialog-custom-report.c:487
 #, fuzzy
```
Comment (fellen) [inline po/fr.po:1678]: @thjungers starting from here you forgot to remove some fuzzy flags.

**Comment by fellen** [comment]
@TianXing-Yi (or the conributor without github account) can  you check this:
```
msgfmt -c --check-accelerators="_" --statistics po/zh_CN.po 
po/zh_CN.po:2885: msgstr lacks the keyboard accelerator mark '_'
po/zh_CN.po:14307: msgstr lacks the keyboard accelerator mark '_'
msgfmt: found 2 fatal errors
3156 translated messages, 1605 fuzzy translations, 718 untranslated messages.

```

**Comment by fellen** [comment]
Thanks!

**Comment by TianXing-Yi** [comment]
> @TianXing-Yi (or the conributor without github account) can  you check this:
> 
> ```
> 
> msgfmt -c --check-accelerators="_" --statistics po/zh_CN.po 
> 
> po/zh_CN.po:2885: msgstr lacks the keyboard accelerator mark '_'
> 
> po/zh_CN.po:14307: msgstr lacks the keyboard accelerator mark '_'
> 
> msgfmt: found 2 fatal errors
> 
> 3156 translated messages, 1605 fuzzy translations, 718 untranslated messages.
> 
> 
> 
> ```

Just have modified them.

**Pull request merged**